packages/list-box/src/vaadin-list-box-core-styles.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/component-base/src/style-props.js';
```

Comment:
I suppose this is not required as an import.

Reply:
My bad, removed.